examples/exec/NuGet.config

```diff
@@ -0,0 +1,5 @@
+<configuration>
```

Comment:
Do we want to add this file?

Reply:
If you want the 'F5' experience to work with this example, then yes.

`ClientWebSocket` only works properly with corefx 2.1 which has dotnet/corefx#25645; so very recent nightly builds. If you don't target `netcoreapp2.1` the code will compile but you'll hit a bug in .NET Core so the sample won't work.

Once there's a public preview of .NET Core 2.1 this can be removed.

Reply:
Well, turns out Travis doesn't have .NET Core 2.1 (which is reasonable 😄 ), this caused the build to break so I reverted this change for now.